packages/experience/src/Providers/UserInteractionContextProvider/UserInteractionContext.tsx

```diff
@@ -10,6 +12,23 @@ export type UserInteractionContextType = {
   // The sso connectors that are enabled for the current domain
   ssoConnectors: SsoConnectorMetadata[];
   setSsoConnectors: React.Dispatch<React.SetStateAction<SsoConnectorMetadata[]>>;
+  identifierInputValue?: IdentifierInputValue;
+  setIdentifierInputValue: React.Dispatch<React.SetStateAction<IdentifierInputValue | undefined>>;
+  forgotPasswordIdentifierInputValue?: IdentifierInputValue;
+  setForgotPasswordIdentifierInputValue: React.Dispatch<
```

Comment:
nit: for future references, better add some comments to indicate the use case of these cached fields.